build.gradle

```diff
@@ -1,10 +1,21 @@
+buildscript {
+    version = "0.21.0"
+}
+
 plugins {
     id 'java'
     id 'jacoco'
     id 'com.auth0.gradle.oss-library.java'
 }
 group = 'com.auth0'
 
+def signingKey = findProperty('SIGNING_KEY')
+def signingKeyPwd = findProperty('SIGNING_PASSWORD')
+
+signing {
+    useInMemoryPgpKeys(signingKey, signingKeyPwd)
+}
```

Comment:
Signing is now happening in memory, using Circle CI environment variables for the Key and Password.